docs/src/reference/python/operations/sum_samples.rst

```diff
@@ -0,0 +1,4 @@
+lstsq
+======
```

Comment:
```suggestion
sum_samples
===========
```

Reply:
I was thinking of doing now, I thought it would not take much but I couldn't think an immediate way to change torch._index _add for the computation of mean. 
for numpy I think you can simply do np.mean.at